code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm

```diff
@@ -3302,8 +3302,178 @@
+	glass_desc = "It tastes like plain berry juice. Is this supposed to be alcoholic?"
+
+//Kaed's Unathi Cocktails
+//=======
+//What an exciting time we live in, that lizards may drink fruity girl drinks.
+/datum/reagent/butanol/moghesmargarita
+	name = "Moghes Margarita"
+	id = "moghesmargarita"
+	description = "A classic human cocktail, now ruined with cactus juice instead of tequila."
+	color = "#8CFF8C"
+	strength = 30
+	taste_description = "lime juice"
+
+	glass_icon_state = "cactusmargarita"
+	glass_name = "glass of Moghes Margarita"
+	glass_desc = "A classic human cocktail, now ruined with cactus juice instead of tequila."
+	glass_center_of_mass = list("x"=16, "y"=8)
+
+/datum/reagent/butanol/cactuscreme
+	name = "Cactus Creme"
+	id = "cactuscreme"
+	description = "A tasty mix of berries and cream with xuizi juice, for the discerning unathi."
+	color = "#664300"
+	strength = 15
+	taste_description = "creamy berries"
+
+	glass_icon_state = "cactuscreme"
+	glass_name = "glass of Cactus Creme"
+	glass_desc = "A tasty mix of berries and cream with xuizi juice, for the discerning unathi."
+	glass_center_of_mass = list("x"=17, "y"=8)
+
+/datum/reagent/butanol/bahamalizard
+	name = "Bahama Lizard"
+	id = "bahamalizard"
+	description = "A tropical cocktail containing cactus juice from Moghes, but no actual alcohol."
+	color = "#FF7F3B"
+	strength = 15
+	taste_description = "lime and orange"
+
+	glass_icon_state = "bahamalizard"
+	glass_name = "glass of Bahama Lizard"
+	glass_desc = "A tropical cocktail containing cactus juice from Moghes, but no actual alcohol."
+	glass_center_of_mass = list("x"=16, "y"=5)
+
+/datum/reagent/butanol/lizardphlegm
+	name = "Lizard Phlegm"
+	id = "lizardphlegm"
+	description = "Looks gross, but smells fruity."
+	color = "#8CFF8C"
+	strength = 20
+	taste_description = "creamy fruit"
+
+	glass_icon_state = "lizardphlegm"
+	glass_name = "glass of Lizard Phlegm"
+	glass_desc = "Looks gross, but smells fruity."
+
+/datum/reagent/butanol/cactustea
+	name = "Cactus Tea"
+	id = "cactustea"
+	description = "Tea flavored with xuizi juice."
+	color = "#664300"
+	strength = 10
+	taste_description = "tea"
+
+	glass_icon_state = "icepick"
+	glass_name = "glass of Cactus Tea"
+	glass_desc = "Tea flavored with xuizi juice."
+
+/datum/reagent/butanol/moghespolitan
+	name = "Moghespolitan"
+	id = "moghespolitan"
+	description = "Pomegranate syrup and cactus juice, with a splash of Sarezhi Wine. Delicious!"
+	color = "#664300"
+	strength = 27
+	taste_description = "fruity sweetness"
+
+	glass_icon_state = "moghespolitan"
+	glass_name = "glass of Moghespolitan"
+	glass_desc = "Pomegranate syrup and cactus juice, with a splash of Sarezhi Wine. Delicious!"
+
+/datum/reagent/butanol/wastelandheat
+	name = "Wasteland Heat"
+	id = "wastelandheat"
+	description = "A mix of spicy cactus juice to warm you up."
+	color = "#664300"
+	strength = 40
+	adj_temp = 60
+	targ_temp = 390
```

Comment:
3395-3396: Travis is claiming these two lines are problematic.


> code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm:3395:error: adj_temp: undefined var
> code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm:3396:error: targ_temp: undefined var

Not sure why seeing as how similar variables have been used in this same file here.

Reply:
According to Lohikar, those variables are not defined for butanol like they are for ethanol.

This is apparently fixed in his recent reagents refactor, and he's told me I only need to wait until that goes through to implement my drink PR.

Reply:
I see.